assets/js/src/common/remove_wrap_margin/remove_wrap_margin.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const RemoveWrapMargin = () => (
+  <>
+    {/* eslint-disable-next-line react/no-danger */}
+    <style dangerouslySetInnerHTML={{ __html: '.wrap { margin: 0 !important; }' }} />
+  </>
+);
+
+export default RemoveWrapMargin;
```

Comment:
I was thinking if there could be a more elegant solution. I think we can use useLayoutEffect and its cleanup function in parent component and actually use an HTML class instead of style tag.
Something like: 
```
useLayoutEffect(() => {
  const wrap = document.getElementsByClassName('wrap')[0];
  wrap.classList.add('wrap-no-margin');
  return () => { wrap.classList.remove('wrap-no-margin') }
})

In campaign_stats/page.tsx should do the job. I find it slightly better since it would allow us having CSS in one place. But it is not a big deal.

Reply:
Good thinking. We already use a couple of these components, so we better unify all of them. I'll wait for Alex to come back from vacation and start a discussion in Slack on how to write them better.